### ISSUE TITLE
BtK doesn't create correct aliases in hosts file

### ISSUE THREAD

**Issue by dmikov** [microsoft/mindaro#205]
**Describe the bug**
When trying to forward api pod, all related services are getting mapped in the  hosts. Unfortunately for some reason the MongoDb pods from replicaset getting wrong aliases, so api pod crashes with time out. 
Here is hosts file:
```
# Added by Bridge To Kubernetes
127.1.1.8 swimlane-sw-mongo swimlane-sw-mongo.default swimlane-sw-mongo.default.svc swimlane-sw-mongo.default.svc.cluster.local
127.1.1.7 swimlane-sw-mongo-1.swimlane-sw-mongo swimlane-sw-mongo-1.swimlane-sw-mongo.svc swimlane-sw-mongo-1.swimlane-sw-mongo.svc.cluster.local
127.1.1.6 swimlane-sw-mongo-0.swimlane-sw-mongo swimlane-sw-mongo-0.swimlane-sw-mongo.svc swimlane-sw-mongo-0.swimlane-sw-mongo.svc.cluster.local
127.1.1.5 swimlane-sw-mongo-2.swimlane-sw-mongo swimlane-sw-mongo-2.swimlane-sw-mongo.svc swimlane-sw-mongo-2.swimlane-sw-mongo.svc.cluster.local
127.1.1.4 sw-api sw-api.default sw-api.default.svc sw-api.default.svc.cluster.local
127.1.1.3 kubernetes kubernetes.default kubernetes.default.svc kubernetes.default.svc.cluster.local
127.1.1.2 sw-reports sw-reports.default sw-reports.default.svc sw-reports.default.svc.cluster.local
127.1.1.1 sw-web sw-web.default sw-web.default.svc sw-web.default.svc.cluster.local
# End of section
```
Notice that other services have domain part `.default.svc.cluster.local`, but mongo pods `.svc.cluster.local`, so when driver tries to talk to `swimlane-sw-mongo-2.swimlane-sw-mongo.default.svc.cluster.local` it cannot find it. Unfortunately this is what replica set recognizes itself as internally.
 
**To Reproduce**
Steps to follow to reproduce this issue.

 
**Expected behavior**
 
**Logs**
Attach logs from the following directory:
For Windows: %TEMP%/Bridge to Kubernetes
For OSX/Linux: $TMPDIR/Bridge to Kubernetes
If you are a Visual Studio user,  
- Set the environment variable MS_VS_AZUREDEVSPACES_TOOLS_LOGGING_ENABLED=true
- Open Visual Studio and run your scenario
- Provide logs from: %temp%\Microsoft.VisualStudio.Kubernetes.Debugging
 
**Environment Details**
Client used (VS Code/Visual Studio):
 
Client's version:
Operating System:
 
**Additional context**
Add any other outputs from the clients or context you would like to share.

**Comment by dmikov** [comment]
[bridge-endpointmanager-2021-07-15-11-55-36-674137.txt](https://github.com/microsoft/mindaro/files/6823536/bridge-endpointmanager-2021-07-15-11-55-36-674137.txt)

**Comment by lolodi** [comment]
@dmikov Thanks for reporting this issue!
We just discovered the problem fixed it. The fix is going to be part of the next release probably next week.

**Comment by dmikov** [comment]
Any update on release?

**Comment by lolodi** [comment]
Hi @dmikov,
we had some slow down in in this latest release so it should be out in a week or so. Thanks for your patience!
If you'd like, you can use our staging bits in the mean time following these instructions:
https://github.com/microsoft/mindaro/issues/195#issuecomment-872663727

**Comment by dmikov** [comment]
I downloaded new version, but the problem still there.

**Comment by daniv-msft** [comment]
@dmikov Yes, the new VS Code version released recently only makes our new changes available to ~10% of the customers, so that we can detect issues without pushing the new release to 100% immediately. We did detect an issue with these 10% (binaries signing issue for some Mac customers), and so we are fixing this before pushing to more customers.
If you want to try the fix @lolodi mentioned above, please feel free to try our staging bits as he suggests: it's quite easy to do, and you will get our latest version there.
